doc/how_to/auto_curation_training.rst

```diff
@@ -0,0 +1,58 @@
+How to train a model to predict curation labels
```

Comment:
What do you think of merging the "How to train/How to use" into a "How to train and use ..."